src/main/java/com/sngular/annotation/processor/mapping/BigDecimalArrayMapping.java

```diff
@@ -0,0 +1,27 @@
+package com.sngular.annotation.processor.mapping;
+
+import com.sngular.annotation.processor.model.FieldValidations;
+import org.apache.commons.rng.UniformRandomProvider;
+
+public class BigDecimalArrayMapping implements TypeMapping<String> {
```

Comment:
Why TypeMapping<String> should not be TypeMapping<BigDecimal> or TypeMapping<Number>?